core/src/main/java/hudson/scm/ChangeLogParser.java

```diff
@@ -52,6 +63,16 @@
         }
     }
 
+    /**
+     * Method for compatibility with AbstractBuild.
```

Comment:
A little confused on how AbstractBuild will behave when it runs [this](https://github.com/hathoward/jenkins/blob/13f25160d4f662641a2b449315a890e0207d376d/core/src/main/java/hudson/model/AbstractBuild.java#L861).

If a SCM plugin only overrides the [Run based parse function](https://github.com/hathoward/jenkins/blob/13f25160d4f662641a2b449315a890e0207d376d/core/src/main/java/hudson/scm/ChangeLogParser.java#L47) will AbstractBuild run it or still revert to [AbstractBuild based parse function](https://github.com/hathoward/jenkins/blob/13f25160d4f662641a2b449315a890e0207d376d/core/src/main/java/hudson/scm/ChangeLogParser.java#L72)?